tests/health.test.ts

```diff
@@ -0,0 +1,18 @@
+import request from 'supertest';
+import { Express } from 'express-serve-static-core';
+import { getServer, startMongoConnection } from "../app";
+
+let server: Express
+
+beforeAll(async () => {
+  startMongoConnection();
+  server = getServer();
+})
+
+
+describe("GET /health", () => {
```

Comment:
I thought this was part of the previous branch... Or has that not been merged yet?